api/provider/aws/service_delete.go

```diff
@@ -1,5 +1,65 @@
 package aws
 
-func (s *ServiceProvider) Delete(ServiceID string) error {
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+)
+
+// Delete a specified service within a cluster.
+func (s *ServiceProvider) Delete(serviceID string) error {
```

Comment:
needs to take an environmentID as a param as well

Reply:
the top level function's done take the environmentID as a param

Reply:
So keep this line https://github.com/quintilesims/layer0/blob/51fc6a03f3dffbb5ac38dc81cc94f1f271e8cc0c/api/provider/aws/service_delete.go#L13

Reply:
yep